style.css

```diff
@@ -4,8 +4,54 @@
   box-sizing: border-box;
 }
 
+h2 {
+  border-top: 1.6px solid black;
+  padding-top: 20px;
+}
+
+li {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  border: 3px solid #000;
+  align-items: center;
+  padding: 12px;
+  width: 90%;
+  margin: 0 auto;
+  margin-top: 20px;
+}
+
+.nav h2 {
+  border: none;
+  font-size: 28px;
+  padding: 1%;
+}
+
+.nav ul {
+  display: flex;
+  width: 40%;
+}
+
+.nav ul li {
+  list-style-type: none;
+  border-right: 3px solid #000;
```

Comment:
- [ ] If we follow the given wireframe as a guide you need to remove the `right bar` of the `contact `section. Kindly remove it. Take a look at your screenshot below 👇 
 
![Screenshot (178)_LI](https://user-images.githubusercontent.com/67087664/153393212-5e126bae-0d70-450a-8c2c-8c35f39067c7.jpg)

Reply:
Aah okay. done